EntAppProject/src/main/java/com/jobook/model/Address.java

```diff
@@ -1,19 +1,19 @@
-package com.jobook.EntAppProject.model;
+package com.jobook.model;
 
 import org.springframework.stereotype.Component;
 
 public class Address {
-	private String city, state, zip, line1, line2;
-	private Integer id;
+	private String city, state, zip, lineOneDescription, lineTwoDescription;

```

Comment:
Renamed ID, line1, and line2. I was unsure what these stood for, so adding a descriptive name can be helpful (or comments).